src/Sql/dbo/Stored Procedures/OrganizationUser_DeleteById.sql

```diff
@@ -17,6 +17,18 @@ BEGIN
     WHERE
         [Id] = @Id
 
+    UPDATE c
```

Comment:
🎨 I'd prefer to add a comment here clarifying what this block is doing.

Reply:
Maybe this logic could be extracted to a new sproc since its duplicated in `OrganizationUser_DeleteByIds`?